### ISSUE TITLE
Unable to get 'ExtendedProperty' of undefined or null refrence when in thumbnail view

### ISSUE THREAD

**Issue by duarteframos** [tablacus/TablacusExplorer#736]
For the longest time I've been getting this scripting error whenever I use thumbnails view in both my work and home computer.
It always happens in folders with images on them when I use one of the thumbnails view sizes. It seems harmless and if I click to continue no other issues seem to occur, it is just a small annoyance.

![20240321_092214](https://github.com/tablacus/TablacusExplorer/assets/8735426/f470350e-ff58-4364-9d97-3e0b28b06ea6)

Can this be fixed? 

```
TE64 24.2.27 Microsoft Windows 10 Pro 64-bit 22H2 (10.0.19045) IE/11 JS/11.0.16384 pt_pt 96 Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz Windows Defender
```

```
mouse 1.33,key 1.26,mainmenu 1.16,toolbar 1.44,tabplus 1.97,tabcolorplus 1.07,tabgroups 1.45,fixedtabname 1.00,sidetreeview 1.32,download 1.08,extract 1.15,innerfilterbar 1.34,innerbreadcrumbsaddressbar 1.50,innerback 1.14,innerup 1.10,innerforward 1.16,innersearchbar 1.13,innerrefresh 1.07,flat 1.23,favoritesbar 1.43,split 1.26,undoclosetab 1.16,titlebar 1.09,segoeicons 1.04,xfinder 1.31,filtericon 1.11,usercss 1.08,gohome 1.04,sethome 1.04,foldersettings 1.39,quickmenu 1.17,switchhidden 1.05,switchhiddenfilter 1.03,switchprevpane 1.03,switchpane 1.05,inactivepane 1.11,tooltippreview 1.28,preview 1.47,label 1.45,overlaylabel 1.01,folderlistmenu 1.15,clipboardimage 1.01,sizestatus 1.27,countbar 1.12,labelgroups 1.03,pathcolumn 1.02,synchronize 1.06,disabletreedrop 1.04,treeviewfilter 1.06,checkbox 1.13,forcerefresh 1.10,tabposition 1.10,addonsupdater 1.10,syncselect 1.03,multiprocess 1.37,wlx 1.14
```

**Comment by ccfs** [comment]
This problem is probably in one (or more) of these add-ons:

- preview
- tooltippreview
- pathcolumn
- sizestatus
- syncselect
- flat

Please try to disable one (or more) add-on (especially preview and tooltippreview) and see, so that I can narrow down the problem area.

Besides,
1. What do you mean by "thumbnail view"? Large icon view?
2. How do you get the error? Just after TE start up? When you switch to "thumbnail view"? When you move to a particular folder? When you move to a particular file? Or what?
3. Please give more details about how you get the error, e.g. steps and a full screen dump.

**Comment by ccfs** [comment]
After some attempts, I have got the error by changing an option in Preview add-on.
Please overwrite script.js of Preview add-on with [this](https://github.com/tablacus/TablacusExplorer/files/14720061/script.zip).

**Comment by duarteframos** [comment]
>1. What do you mean by "thumbnail view"? Large icon view?

It seems to happen in whichever of the views that generates thumbnails of the image, rather than a filetype icon.
Extra Large icons, Large icons or Medium Icons, as far as I can tell

> 2. How do you get the error? Just after TE start up? When you switch to "thumbnail view"? When you move to a particular folder? When you move to a particular file? Or what?

I couldn't pinpoint an exact situation, it happens "around" folders with images in them. Most of the time it happens when I navigate into a folder with images, but it also happens when I'm already seeing its content and change the view mode, refresh, or select/deselect some files.

For example right now I was testing, and it happened when I navigated away from a folder with images up to the parent folder. 

> After some attempts, I have got the error by changing an option in Preview add-on. Please overwrite script.js of Preview add-on with [this](https://github.com/tablacus/TablacusExplorer/files/14720061/script.zip). Besides, please turn OFF "Extract" which seems to have a problem with media files.

I'm giving it a try as we speak. From my limited immediate testing I'm no longer see any error dialogs, but I'll give it a run through the whole day and see how it goes. 

Thanks for the patch, so far it seems promising. I'll report back later with more thorough testing.

**Comment by tablacus** [comment]
I have fixed it on preview add-on v1.48.
Please try to update it.

Regards,
